webpack.config.js

```diff
@@ -222,11 +224,15 @@ const config = {
         },
       },
       {
-        test: /\.(jpe?g|png|gif|svg|mp4)$/i,
+        test: /\.(jpe?g|png|gif|mp4)$/i,
         use: {
           loader: 'url-loader',
         },
       },
+      {
+        test: /\.svg$/,
+        use: ['@svgr/webpack'],
+      },
```

Comment:
In `src/app/components/text/infobox.js`, it uses the svg file (`src/app/static/infobox-log.svg`) as React component.
I haven't changed the way how we use svg files.
Thus, I add webpack svg loader here to build the webpack bundles for now.
If we have clear plan about how to use svg files, we can remove this loader.